settings.py

```diff
@@ -1560,8 +1557,6 @@ def _pkce_required(client_id):
     ('userreports', META_DB),
     ('custom_data_fields', META_DB),
     # needed to make couchdbkit happy
```

Comment:
is this comment still relevant?

Reply:
No. e2937409b11be6d22d74ce070b8ee5c2669f294d